lib/convection/model/template/resource/aws_efs_mount_target.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        # Represents an {https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-efs-mounttarget.html
+        # Amazon EFS Mount Target}
+        class EFSMountTarget < Resource
+          type 'AWS::EFS::MountTarget'
+          property :file_system_id, 'FileSystemId'
+          property :ip_address, 'IpAddress'
+          property :security_groups, 'SecurityGroups', :type => :list
```

Comment:
nevermind, it looks like this is a list of strings not a list of other AWS types, so it should be fine as-is